From b1d672d3b14be78e38218f3b8ccdc6494b41a346 Mon Sep 17 00:00:00 2001
From: Hao Lan <lanhao@huawei.com>
Date: Fri, 9 Dec 2022 11:01:58 +0800
Subject: [PATCH] net: hns3: fix the HCLGE_OPC_WOL_CFG opcode id for wol
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: b1d672d3b14be78e38218f3b8ccdc6494b41a346
Modified-by-SEL: No


driver inclusion
category: bugfix
bugzilla: https://gitee.com/openeuler/kernel/issues/I65FSF
CVE: NA

----------------------------------------------------------------------

Fix the HCLGE_OPC_WOL_CFG opcode id for wol.

Fixes: c3c5f044b7dc ("net: hns3: support wake on lan configuration and query")
Signed-off-by: Hao Lan <lanhao@huawei.com>
Signed-off-by: Jiantao Xiao <xiaojiantao1@h-partners.com>
(cherry picked from commit 987f422a346e84c9d29a7a83d4a8448a02bad97c)
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 .../net/ethernet/hisilicon/hns3/hns3_common/hclge_comm_cmd.h    | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/hisilicon/hns3/hns3_common/hclge_comm_cmd.h b/drivers/net/ethernet/hisilicon/hns3/hns3_common/hclge_comm_cmd.h
index 2fde935dcbbd..c7146256f721 100644
--- a/drivers/net/ethernet/hisilicon/hns3/hns3_common/hclge_comm_cmd.h
+++ b/drivers/net/ethernet/hisilicon/hns3/hns3_common/hclge_comm_cmd.h
@@ -294,8 +294,8 @@ enum hclge_opcode_type {
 	HCLGE_PPP_CMD0_INT_CMD		= 0x2100,
 	HCLGE_PPP_CMD1_INT_CMD		= 0x2101,
 	HCLGE_MAC_ETHERTYPE_IDX_RD      = 0x2105,
-	HCLGE_OPC_WOL_CFG		= 0x2200,
 	HCLGE_OPC_WOL_GET_SUPPORTED_MODE	= 0x2201,
+	HCLGE_OPC_WOL_CFG		= 0x2202,
 	HCLGE_NCSI_INT_EN		= 0x2401,
 
 	/* ROH MAC commands */
-- 
2.33.0

